Python-Function-ID.md

```diff
@@ -0,0 +1,42 @@
+# Python id(object)
+
+`id()` is a built-in function in Python 3, which returns the *identity* of an object. The *identity* is a unique integer for that object during its lifetime. In CPython implementation, this is the address of the object in memory which is not necessarily true for other python implementations.
```

Comment:
> In CPython implementation

A beginner would want to know what you mean by _CPython implementation_, so better link to the GitHub mirror of CPython interpreter, or some article that talks about this.